composer.json

```diff
@@ -28,11 +29,12 @@
         "corley/influxdb-sdk": "^0.5.1",
         "doctrine/dbal": "~2.0",
         "jimdo/prometheus_client_php": "^0.5",
-        "kriswallsmith/buzz": "*",
         "okitsu/zabbix-sender": "*@dev",
         "symfony/config": "~3.3 || ~4.0",
         "symfony/dependency-injection": "~3.3 || ~4.0",
-        "symfony/http-kernel": "~3.3 || ~4.0"
+        "symfony/http-kernel": "~3.3 || ~4.0",
+        "symfony/http-client": "^4.0 || ^5.0",
+        "kriswallsmith/buzz": "^0.16"
```

Comment:
^0.16|^0.17|^1
no need to change the order

Reply:
^1 requires PSR17 request factory in the constructor, so i don't think it would work here, unless we add a PSR17 package to composer, however, we will end up breaking BC we existing code that uses buzz 0.* already.